apps/omg_eth/lib/omg_eth/transact.ex

```diff
@@ -0,0 +1,61 @@
+# Copyright 2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Eth.Transact do
+  @moduledoc """
+  An interface to Ethereum client transact function.
+  """
+  require Logger
+  import OMG.Eth.Encoding, only: [from_hex: 1]
+
+  @doc """
+  Send transaction to be singed by a key managed by Ethereum node, geth or parity.
```

Comment:
I know you need a karaoke but...

```suggestion
  Send transaction to be signed by a key managed by Ethereum node, geth or parity.
```